specification/components/schemas/DocumentReference.yaml

```diff
@@ -112,3 +112,26 @@ properties:
               type: string
               format: date
               example: '2021-04-22'
+            extension:
+              type: array
+              minItems: 1
+              maxItems: 1
```

Comment:
Do we always include the `attachedBy` extension? Just looking at the FHIR profile (https://fhir.nhs.uk/STU3/StructureDefinition/eRS-DocumentReference-1) it looks like `attachedBy` is optional.